inspections/codesniffer/phpcs.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+function remove_diff_range {
+  sed 's/:[0-9][0-9]*-[0-9][0-9]*$//' | sort | uniq
+}
+
+function lint_php_files {
+	PROJECT_ROOT="$(git rev-parse --show-toplevel)"
+	TEMP_DIRECTORY=$(mktemp -d 2>/dev/null || mktemp -d -t 'dev-lib')
+
+	DIFF_BASE=${DIFF_BASE:-origin/master}
+	DIFF_HEAD=${DIFF_HEAD:-$(git rev-parse HEAD)}
+	DIFF_ARGS="$DIFF_BASE...$DIFF_HEAD"
+
+	git diff --diff-filter=AM --no-prefix --unified=0 "$DIFF_ARGS" | \
+		php /parse-diff-ranges.php | \
+		{ grep -E '\.php(:|$)' || true; } > "$TEMP_DIRECTORY/paths-scope-php"
+
+	WPCS_DIR=${WPCS_DIR:-/tmp/wpcs}
+	WPCS_GITHUB_SRC=${WPCS_GITHUB_SRC:-WordPress-Coding-Standards/WordPress-Coding-Standards}
+	WPCS_GIT_TREE=${WPCS_GIT_TREE:-master}
+	WPCS_STANDARD=${WPCS_STANDARD:-WordPress-Core}
+	WordPressVIPMinimum=${WordPressVIPMinimum:-/tmp/VIP}
+
+	git clone -b "$WPCS_GIT_TREE" "https://github.com/$WPCS_GITHUB_SRC.git" "$WPCS_DIR" > /dev/null 2>&1
+	git clone -b master "https://github.com/Automattic/VIP-Coding-Standards.git" "$WordPressVIPMinimum"
+	ln -s ${WordPressVIPMinimum}/WordPressVIPMinimum ${WPCS_DIR}/WordPressVIPMinimum
+	ln -s ${WordPressVIPMinimum}/WordPress-VIP-Go ${WPCS_DIR}/WordPress-VIP-Go
+	pushd "$WPCS_DIR" > /dev/null 2>&1
+	git clone https://github.com/wimg/PHPCompatibility /tmp/phpcompat > /dev/null 2>&1
```

Comment:
We need to add `PHPCompatibilityWP` but it's not a blocker.

Reply:
Added PHPCompatibilityWP.

Reply:
PHPCompatibilityWP has dependency and can't run out of the box using git like that - https://github.com/PHPCompatibility/PHPCompatibilityWP/blob/master/composer.json#L21-L22 @mrrobot47

Reply:
Ref: #11

Reply:
Updated to composer global require method for setup of phpcs entirely to fix this issue.